src/test/java/org/apache/maven/shared/invoker/MavenCommandLineBuilderTest.java

```diff
@@ -109,7 +98,7 @@ public void testShouldFailToSetLocalRepoLocationFromRequestWhenItIsAFile()
     @Test
     public void testShouldSetLocalRepoLocationGlobally() throws IOException
     {
-        File lrd = temporaryFolder.newFolder( "workdir" ).getCanonicalFile();
+        File lrd = Files.createDirectory(temporaryFolder.resolve("workdir")).toFile().getCanonicalFile();
```

Comment:
Please follow: https://maven.apache.org/developers/conventions/code.html in whole file.

Should be:
```suggestion
        File lrd = Files.createDirectory( temporaryFolder.resolve( "workdir" ) ).toFile().getCanonicalFile();
```

missing spaces